from main import ftb
ftb(filename="payment.py", line="128", errorin="pay_process", error="PaymentFailed", errortext="Bad Internet connection occurs payment wasn't done.")
